projects/premake/premake5.lua

```diff
@@ -1,164 +1,182 @@
-workspace "wren-cli"

```

Comment:
Let's fix up the line endings or whatever has caused this entire file to be a diff :)